dataactvalidator/scripts/get_city_file.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import os
+import requests
+import logging
+import re
+import boto3
+import urllib.request
+from bs4 import BeautifulSoup
+import zipfile
+import datetime
+
+from dataactcore.config import CONFIG_BROKER
```

Comment:
I think decoupling scripts is a bigger conversation that needs to be had. As of now, all scripts are connected because we do have useful things like having a standardized logging config and using the environment variables. Some of them (like the path) are useful in multiple ways. I'll defer to @dpb-bah to determine if this is something we should do for this PR

Reply:
Agreed, I'm agree with most of these changes as a fan of the decoupling effort but it can hold till we approach the epic.